.pre-commit-config.yaml

```
@@ -15,3 +15,36 @@ repos:
         additional_dependencies:
           - prettier
           - prettier-plugin-toml
+
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.12.0
+    hooks:
+      - id: isort
+        args: ["--filter-files"]
+
+  - repo: https://github.com/psf/black
+    rev: 23.1.0
+    hooks:
+      - id: black
+
+  - repo: https://github.com/pycqa/flake8
+    rev: 6.0.0
+    hooks:
+      - id: flake8
+        additional_dependencies:
+          - flake8-2020 >= 1.6.0
+          - flake8-isort >= 4.1.1
+          - darglint
+          - flake8-docstrings # uses pydocstyle
+
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.1.1
+    hooks:
+      - id: mypy
+        additional_dependencies:
+          - types-PyYAML
+
+  - repo: https://github.com/pycqa/pylint.git
+    rev: v2.17.0
+    hooks:
+      - id: pylint
```

This should help pylint out, not sure how I missed it the first time.
```suggestion
      - id: pylint
        additional_dependencies:
          - PyYAML
```